Engines/Wine/Verbs/Remove Mono/script.js

```diff
@@ -3,12 +3,12 @@ include("utils.functions.filesystem.files");
 include("engines.wine.plugins.regedit");
 
 /**
-* Verb to remove mono
-* @returns {Wine} Wine object
-*/
+ * Verb to remove mono
+ * 
+ * @returns {Wine} Wine object
+ */
 Wine.prototype.removeMono = function () {
-    if (this.uninstall("Mono"))
-    {
+    if (this.uninstall("Mono")) {
```

Comment:
Why doesn't ESLint handle this?

Reply:
I don't know. I performed the formatting in this PR with Visual Studio Code.